docs/ibc/middleware/integration.md

```diff
@@ -0,0 +1,69 @@
+<!--
+order: 2
+-->
+
+# Integrating IBC Middleware into a Chain
+
+Learn how to integrate IBC middleware(s) with a base application to your chain. The following document only applies for SDK chains.
```

Comment:
Should we explicitly say "Cosmos SDK" at the top of the doc at least once, then later references to "SDK" can be inferred? Feel free to ignore, just a small nit! 

```suggestion
Learn how to integrate IBC middleware(s) with a base application to your chain. The following document only applies for Cosmos SDK chains.
```